samples/javascript_nodejs/57.teams-conversation-bot/bots/teamsConversationBot.js

```diff
@@ -7,61 +7,114 @@ const {
+            //  console.log(context);
             context.activity.membersAdded.forEach(async (teamMember) => {
                 if (teamMember.id !== context.activity.recipient.id) {
-                    await context.sendActivity(`Welcome to the team ${ teamMember.givenName } ${ teamMember.surname }`);
+                    await context.sendActivity(
+                        `Welcome to the team ${teamMember.givenName} ${teamMember.surname}`
+                    );
                 }
             });
             await next();
         });
+
+        //Subscribe to different conversation events
+
+        this.onTeamsChannelCreatedEvent(async (context, next) => {
+            await this.onTeamsChannelCreated(context);
+            await next();
+        });
+
+        this.onTeamsChannelRenamedEvent(async (context, next) => {
+            await this.onTeamsChannelRenamed(context);
+            await next();
+        });
+        this.onTeamsChannelDeletedEvent(async (context, next) => {
+            await this.onTeamsChannelDeleted(context);
+            await next();
+        });
+        this.onTeamsChannelRestoredEvent(async (context, next) => {
+            await this.onTeamsChannelRestored(context);
+            await next();
+        });
+
+        this.onTeamsTeamRenamedEvent(async (context, next) => {
+            await this.onTeamsTeamRenamed(context);
+            await next();
+        });
+
+        this.onReactionsAdded(async (context) => {
+            const reactionsAdded = context.activity.reactionsAdded;
+            if (reactionsAdded && reactionsAdded.length > 0) {
+                for (let i = 0; i < reactionsAdded.length; i++) {
+                    const reaction = reactionsAdded[i];
+                    const newReaction = `You reacted with '${reaction.type}' to the following message: '${context.activity.replyToId}'`;
+                    const resourceResponse = context.sendActivity(newReaction);
+                    // Save information about the sent message and its ID (resourceResponse.id).
+                }
+            }
+        });
+
+        this.onReactionsRemoved(async (context) => {
+            const reactionsRemoved = context.activity.reactionsRemoved;
+            if (reactionsRemoved && reactionsRemoved.length > 0) {
+                for (let i = 0; i < reactionsRemoved.length; i++) {
+                    const reaction = reactionsRemoved[i];
+                    const newReaction = `You removed the reaction '${reaction.type}' from the message: '${context.activity.replyToId}'`;
+                    const resourceResponse = context.sendActivity(newReaction);
+                    // Save information about the sent message and its ID (resourceResponse.id).
+                }
+            }
+        });
     }
 
     async cardActivityAsync(context, isUpdate) {
         const cardActions = [
             {
                 type: ActionTypes.MessageBack,
-                title: 'Message all members',
```

Comment:
please revert to single-quotes for the rest of the file

Reply:
Marking this as Resolved as wrong file went in this PR